src/app/static/src/app/store/modelCalibrate/actions.ts

```diff
@@ -84,8 +84,8 @@ export const actions: ActionTree<ModelCalibrateState, RootState> & ModelCalibrat
                         },
                         {root: true});
                 }
-
                 commit(ModelCalibrateMutation.Calibrated);
+                commit({type: ModelCalibrateMutation.UploadMetadata, payload: data.uploadMetadata});
```

Comment:
I don't think we need to commit the uploadMetadata separately at all - it's part of the main result set (`data`) which is being committed as part of modelRun/RunResultFetched - we could read it straight out of there when needed. We'd just need to add it to the CalibrateResultResponse type.